backend/app/tests/api/routes/documents/test_route_document_permanent_remove.py

```
@@ -36,16 +39,26 @@ def aws_credentials():
+class TestDocumentRouteRemove:
     @openai_responses.mock()
-    def test_permanent_delete_document_from_s3(
+    @patch("app.api.routes.documents.get_provider_credential")
+    @patch("app.api.routes.documents.configure_openai")
+    def test_item_is_soft_removed(
         self,
+        mock_configure_openai,
+        mock_get_credential,
         db: Session,
         route: Route,
         crawler: WebCrawler,
```

Ruff (0.12.2)

Redefinition of unused `crawler` from line 23

(F811)